soap/src/java/com/zimbra/soap/admin/message/GetAccountRequest.java

```diff
@@ -53,6 +53,17 @@
     @XmlAttribute(name=AdminConstants.A_APPLY_COS, required=false)
     private ZmBoolean applyCos = ZmBoolean.ONE /* true */;
 
+    /**
```

Comment:
do we need extra changes to update soap api documentation with this flag? let's create a ticket to release updated soap documentation with this ticket